packages/js/product-editor/src/components/iframe-editor/iframe-editor.scss

```diff
@@ -75,4 +75,10 @@
 			display: none;
 		}
 	}
+
+	.woocommerce-iframe-editor__header-right
+		.interface-pinned-items
+		.components-button {
+		display: flex;
+	}
```

Comment:
Targeted all the component buttons within the iframe header.
Gutenberg targets the specific one's by using `aria-controls` -> https://github.com/WordPress/gutenberg/blob/2465c27c5d541f0ec6ff273a3df1729a49991451/packages/interface/src/components/pinned-items/style.scss#L10
This is added by using an `identifier` but I couldn't get this block settings block to render it, I couldn't figure out why as to my knowledge we are using the components correctly.
Given the above is pretty narrow for the right side of the iframe header I think this will do as well.